fendermint/vm/interpreter/src/chain.rs

```diff
@@ -89,32 +97,65 @@ where
             CheckpointPoolItem::BottomUp(ckpt) => ChainMessage::Ipc(IpcMessage::BottomUpExec(ckpt)),
         });
 
+        // Prepare top down proposals
+        match atomically_or_err::<_, fendermint_vm_topdown::Error, _>(|| {
+            finality_provider.next_proposal()
+        })
+        .await?
```

Comment:
If this fails this will crash the node.

Reply:
Yep, I encountered this during integration testing. I have updated the logic slightly. It will absorb the error due to:
- can always retry in the next block
- other nodes might propose and process successfully